src/debugger/BreakpointManager.js

```diff
@@ -25,6 +25,7 @@ export class BreakpointManager {
   _breakpointMaps: Map<string, PerFileBreakpointMap>;
   _previousStop: StoppedData;
```

Comment:
I do not think you need this anymore.

Reply:
This is updated in onDebuggeeStop and may be used later in other breakpoint logic.